### PR TITLE
Fix Front-end bugs (swiping, uploading, margins)

### LINKED ISSUE #2129: Upload window in v5 disappears before all uploads are finished.
### Detailed description of the problem [REQUIRED]
Upload window in v5 disappears before all uploads are finished. Tested with 30 and more photos. Looks like up to 20 photos it is uploading correctly.
After it disapears looks like photos are still uploaded in the bacground, if you go to Job Logs and click on it every few seconds you can see new photos uploaded but you already exited the album.
If you get back to the album or refresh the website than the "upload" that you don't see is stopped and not all photos are uploaded.

To help me upload photos i enabled: skip_duplicates and use_job_queues but this didn't change the behavior.
Don't know if the issue is related to lychee being hosted behind nginx proxy manager reverse proxy but i have TRUSTED_PROXIES=* env enabled.

### Steps to reproduce the issue
**Steps to reproduce the behavior:**
1. Go to any album
2. Click on upload photos
3. Add around 50 photos for testing

### Output of the diagnostics  [REQUIRED]
*(Settings => Diagnostics or https://example.com/Diagnostics or `php artisan lychee:diagnostics`)*
```
    Diagnostics
    -----------
    Warning: Dropbox import not working. dropbox_key is empty.
    Info: Latest version of PHP is 8.3
    Warning: git (software) is not available.
    Foreign key: access_permissions.user_id     → users.id             : CASCADE
    Foreign key: albums.parent_id               → albums.id            : RESTRICT
    Foreign key: albums.id                      → base_albums.id       : RESTRICT
    Foreign key: albums.cover_id                → photos.id            : CASCADE
    Foreign key: base_albums.owner_id           → users.id             : RESTRICT
    Foreign key: jobs_history.owner_id          → users.id             : RESTRICT
    Foreign key: photos.album_id                → albums.id            : RESTRICT
    Foreign key: photos.owner_id                → users.id             : RESTRICT
    Foreign key: size_variants.photo_id         → photos.id            : RESTRICT
    Foreign key: sym_links.size_variant_id      → size_variants.id     : RESTRICT
    Foreign key: tag_albums.id                  → base_albums.id       : RESTRICT

    System Information
    ------------------
    Lychee Version (git):                    ?? (33354a2) -- Could not compare.
    DB Version:                              5.0.2
    
    composer install:                        --no-dev
    APP_ENV:                                 production
    APP_DEBUG:                               false
    APP_URL:                                 set
    
    System:                                  Linux
    PHP Version:                             8.2.7
    PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
    Timezone:                                Europe/Warsaw
    Max uploaded file size:                  100M
    Max post size:                           100M
    Livewire chunk size:                     12.00 MB
    Max execution time:                      3600
    MySQL Version:                           11.2.2-MariaDB-1:11.2.2+maria~ubu2204
    
    exec() Available:                        yes
    Imagick Available:                       1
    Imagick Enabled:                         1
    Imagick Version:                         1691
    GD Version:                              2.3.3
    Number of foreign key:                   11 found.

    Config Information
    ------------------
    version:                                 050002
    check_for_updates:                       0
    sorting_photos_col:                      taken_at
    sorting_photos_order:                    ASC
    sorting_albums_col:                      max_taken_at
    sorting_albums_order:                    ASC
    imagick:                                 1
    skip_duplicates:                         1
    small_max_width:                         0
    small_max_height:                        360
    medium_max_width:                        1920
    medium_max_height:                       1080
    lang:                                    pl
    image_overlay_type:                      desc
    default_license:                         none
    compression_quality:                     90
    grants_full_photo_access:                1
    delete_imported:                         0
    mod_frame_enabled:                       1
    mod_frame_refresh:                       30
    thumb_2x:                                1
    small_2x:                                1
    medium_2x:                               1
    landing_page_enable:                     0
    site_owner:                              SloniuPl
    landing_title:                           SloniuPl
    landing_subtitle:                        Wesele Sylwi i Bartka
    sm_facebook_url:                         
    sm_flickr_url:                           
    sm_twitter_url:                          
    sm_instagram_url:                        
    sm_youtube_url:                          
    landing_background:                      dist/cat.webp
    site_title:                              Sylwia i Bartek
    footer_show_copyright:                   1
    site_copyright_begin:                    2023
    site_copyright_end:                      2023
    footer_additional_text:                  
    footer_show_social_media:                0
    search_public:                           0
    SL_enable:                               0
    SL_for_admin:                            0
    recent_age:                              1
    grants_download:                         0
    photos_wraparound:                       1
    map_display:                             0
    zip64:                                   1
    map_display_public:                      0
    map_provider:                            OpenStreetMap.org
    force_32bit_ids:                         0
    map_include_subalbums:                   0
    update_check_every_days:                 3
    has_exiftool:                            1
    share_button_visible:                    0
    import_via_symlink:                      0
    has_ffmpeg:                              1
    location_decoding:                       0
    location_decoding_timeout:               30
    location_show:                           0
    location_show_public:                    0
    rss_enable:                              0
    rss_recent_days:                         7
    rss_max_items:                           100
    prefer_available_xmp_metadata:           0
    editor_enabled:                          1
    lossless_optimization:                   0
    swipe_tolerance_x:                       150
    swipe_tolerance_y:                       250
    local_takestamp_video_formats:           .avi|.mov
    log_max_num_line:                        1000
    unlock_password_photos_with_url_param:   0
    nsfw_visible:                            1
    nsfw_blur:                               0
    nsfw_warning:                            0
    nsfw_warning_admin:                      0
    nsfw_banner_override:                    
    map_display_direction:                   1
    album_subtitle_type:                     oldstyle
    upload_processing_limit:                 4
    new_photos_notification:                 0
    legacy_id_redirection:                   1
    zip_deflate_level:                       6
    SA_enabled:                              1
    default_album_protection:                1
    album_decoration:                        layers
    album_decoration_orientation:            row
    allow_username_change:                   1
    auto_fix_orientation:                    1
    use_job_queues:                          1
    random_album_id:                         starred
    use_last_modified_date_when_no_exif_date: 0
    ffmpeg_path:                             /usr/bin/ffmpeg
    ffprobe_path:                            /usr/bin/ffprobe
    layout:                                  justified
    date_format_photo_thumb:                 M j, Y, g:i:s A e
    date_format_photo_overlay:               M j, Y, g:i:s A e
    date_format_sidebar_uploaded:            M j, Y, g:i:s A e
    date_format_sidebar_taken_at:            M j, Y, g:i:s A e
    date_format_hero_min_max:                F Y
    date_format_hero_created_at:             M j, Y, g:i:s A T
    date_format_album_thumb:                 M Y
    upload_chunk_size:                       0
    nsfw_banner_blur_backdrop:               0
    search_pagination_limit:                 1000
    search_minimum_length_required:          4
    photo_layout_justified_row_height:       320
    photo_layout_masonry_column_width:       300
    photo_layout_grid_column_width:          250
    photo_layout_square_column_width:        200
    photo_layout_gap:                        12
    display_thumb_album_overlay:             always
    display_thumb_photo_overlay:             hover

```
### Browser and system
Chrome and Windows 11

### LINKED ISSUE #2151: Swiping L/R no longer works on mobile for v5.0.2
I'm running v5.0.2 on Docker. Most things seem fine, but noticed when browsing on mobile, I can no longer swipe left and right when viewing photos. Instead, I have to tap (twice) to use the L/R buttons. Really quite a jarring change. I am using Firefox for Android. 

Not sure if this is a feature or a bug. Thanks!

### LINKED ISSUE #2207: Uploads over 12MB silently fail, with no errors in log.
### Detailed description of the problem [REQUIRED]

Any photo uploaded over 12MB will not upload completely, and will instead place a file of exactly 12MB in public\livewire-tmp\ and then silently fail with no errors.

Note: Does not happen when `LIVEWIRE_ENABLED=false` is set in `.env`.

### Steps to reproduce the issue
**Steps to reproduce the behavior:**

1. Go to any album
2. Click on upload photos
3. Add any photo over 12MB
4. Upload bar will jump to some point below 100%, Upload box will dissapear (possibly related to [#2129](https://github.com/LycheeOrg/Lychee/issues/2129)?), and no photo will show up on the album.

### Output of the diagnostics  [REQUIRED]
*(Settings => Diagnostics or https://example.com/Diagnostics or `php artisan lychee:diagnostics`)*

```
    Diagnostics
    -----------
        Warning: Dropbox import not working. dropbox_key is empty.
        Info: Latest version of PHP is 8.3
        Warning: Default timezone not properly set; you might experience strange results when importing photos without explicit EXIF timezone
     
    System Information
    ------------------
        Lychee Version (release):                5.0.3
        DB Version:                              5.0.3
        
        composer install:                        --no-dev
        APP_ENV:                                 production
        APP_DEBUG:                               false
        APP_URL:                                 set
        APP_DIR:                                 default
        LOG_VIEWER_ENABLED:                      true
        LIVEWIRE_ENABLED:                        true
        
        System:                                  Linux
        PHP Version:                             8.2.14
        PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
        Timezone:                                UTC
        Max uploaded file size:                  500M
        Max post size:                           500M
        Livewire chunk size:                     12.00 MB
        Max execution time:                      200
        MySQL Version:                           10.5.23-MariaDB-cll-lve
        
        exec() Available:                        yes
        Imagick Available:                       1
        Imagick Enabled:                         1
        Imagick Version:                         1809
        GD Version:                              2.3.3
        Number of foreign key:                   11 found.

    Config Information
    ------------------
        version:                                 050003
        check_for_updates:                       0
        sorting_photos_col:                      taken_at
        sorting_photos_order:                    ASC
        sorting_albums_col:                      max_taken_at
        sorting_albums_order:                    ASC
        imagick:                                 1
        skip_duplicates:                         0
        small_max_width:                         0
        small_max_height:                        360
        medium_max_width:                        1920
        medium_max_height:                       1080
        lang:                                    en
        image_overlay_type:                      desc
        default_license:                         reserved
        compression_quality:                     80
        grants_full_photo_access:                1
        delete_imported:                         0
        mod_frame_enabled:                       1
        mod_frame_refresh:                       30
        thumb_2x:                                1
        small_2x:                                1
        medium_2x:                               1
        landing_page_enable:                     0
        site_owner:                              TobieMutt
        landing_title:                           
        landing_subtitle:                        
        sm_facebook_url:                       
        sm_flickr_url:                     
        sm_twitter_url:              
        sm_instagram_url:
        sm_youtube_url:                         
        landing_background:                   
        site_title:                              TobieMutt
        footer_show_copyright:                   1
        site_copyright_begin:                    2023
        site_copyright_end:                      2024
        footer_additional_text:                  
        footer_show_social_media:                0
        search_public:                           0
        SL_enable:                               0
        SL_for_admin:                            0
        recent_age:                              1
        grants_download:                         1
        photos_wraparound:                       1
        map_display:                             0
        zip64:                                   1
        map_display_public:                      0
        map_provider:                            Wikimedia
        force_32bit_ids:                         1
        map_include_subalbums:                   0
        update_check_every_days:                 3
        has_exiftool:                            0
        share_button_visible:                    0
        import_via_symlink:                      0
        has_ffmpeg:                              0
        location_decoding:                       0
        location_decoding_timeout:               30
        location_show:                           0
        location_show_public:                    0
        rss_enable:                              0
        rss_recent_days:                         7
        rss_max_items:                           100
        prefer_available_xmp_metadata:           0
        editor_enabled:                          1
        lossless_optimization:                   0
        swipe_tolerance_x:                       150
        swipe_tolerance_y:                       250
        local_takestamp_video_formats:           .avi|.mov
        log_max_num_line:                        1000
        unlock_password_photos_with_url_param:   0
        nsfw_visible:                            1
        nsfw_blur:                               0
        nsfw_warning:                            0
        nsfw_warning_admin:                      0
        nsfw_banner_override:                    
        map_display_direction:                   1
        album_subtitle_type:                     oldstyle
        upload_processing_limit:                 4
        new_photos_notification:                 0
        legacy_id_redirection:                   1
        zip_deflate_level:                       6
        SA_enabled:                              1
        default_album_protection:                1
        album_decoration:                        layers
        album_decoration_orientation:            row
        allow_username_change:                   1
        auto_fix_orientation:                    1
        use_job_queues:                          0
        random_album_id:                         starred
        use_last_modified_date_when_no_exif_date: 0
        ffmpeg_path:                             
        ffprobe_path:                            
        layout:                                  justified
        date_format_photo_thumb:                 M j, Y, g:i:s A e
        date_format_photo_overlay:               M j, Y, g:i:s A e
        date_format_sidebar_uploaded:            M j, Y, g:i:s A e
        date_format_sidebar_taken_at:            M j, Y, g:i:s A e
        date_format_hero_min_max:                F Y
        date_format_hero_created_at:             M j, Y, g:i:s A T
        date_format_album_thumb:                 M Y
        upload_chunk_size:                       0
        nsfw_banner_blur_backdrop:               0
        search_pagination_limit:                 1000
        search_minimum_length_required:          4
        photo_layout_justified_row_height:       320
        photo_layout_masonry_column_width:       300
        photo_layout_grid_column_width:          250
        photo_layout_square_column_width:        200
        photo_layout_gap:                        12
        display_thumb_album_overlay:             always
        display_thumb_photo_overlay:             hover
        default_album_thumb_aspect_ratio:        1/1
        use_album_compact_header:                0
        login_button_position:                   left
        back_button_enabled:                     0
        back_button_text:                        Return to Home
        back_button_url:                         /
```

### Browser and system

Firefox and Windows 10

### LINKED ISSUE #2247: Download photo/open in new tab buttons are not working on mobile
### Detailed description of the problem
Buttons to download a photo or to open it in a new tab are not working. They're overlayed by a bar with star, delete, rotate etc. buttons. It happens in both portrait and landscape modes. 

**Screenshots**
image

image

### Browser and system
iPhone 13 mini
iOS 16.3.1
Safari

### LINKED ISSUE #2285: stream_copy_to_stream() HttpException 500
### Detailed description of the problem [REQUIRED]

Upgraded to v5.
While uploading about 50 images, mid way through the process I get this error message in the browser.

In v4, uploads sometimes would fail, but the list would continue and I could see failed uploads in the list. Then I could go and try them again.

```
::stream_copy_to_stream:50 stream_copy_to_stream(): Argument #1 ($from) must be of type resource, null given
App\Image\Files\NativeLocalFile::append:119 stream_copy_to_stream(): Argument #1 ($from) must be of type resource, null given;
```
image

### Output of the diagnostics  [REQUIRED]
```
 Diagnostics
    -----------
        Warning: /ho********************************blic/uploads/ has permissions 0755, but should have 2775
        Warning: /ho********************************blic/uploads/thumb2x has permissions 0775, but should have 2775
        Warning: /ho********************************blic/uploads/thumb2x/56 has permissions 0775, but should have 2775
        Warning: /ho********************************blic/uploads/thumb2x/56/8a has permissions 0775, but should have 2775
        Warning: /ho********************************blic/uploads/thumb2x/56/34 has permissions 0775, but should have 2775
        Warning: 179618 more directories with wrong permissions
        Warning: Dropbox import not working. dropbox_key is empty.
        Warning: You may experience problems when uploading a photo of large size or handling many/large albums. Take a look in the FAQ for details.
        Info: Found 30 small that could be generated.
              You can use `php artisan lychee:generate_thumbs small 30` to generate them.
        Info: Found 19 small2x that could be generated.
              You can use `php artisan lychee:generate_thumbs small2x 19` to generate them.
        Info: Found 44 medium2x that could be generated.
              You can use `php artisan lychee:generate_thumbs medium2x 44` to generate them.
     
    System Information
    ------------------
        Lychee Version (release):                5.1.2
        DB Version:                              5.1.2
        
        composer install:                        --no-dev
        APP_ENV:                                 production
        APP_DEBUG:                               false
        APP_URL:                                 set
        APP_DIR:                                 default
        LOG_VIEWER_ENABLED:                      true
        LIVEWIRE_ENABLED:                        true
        
        System:                                  Linux
        PHP Version:                             8.3.1
        PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
        Timezone:                                America/Vancouver
        Max uploaded file size:                  512M
        Max post size:                           512M
        Livewire chunk size:                     12.00 MB
        Max execution time:                      90
        MySQL Version:                           10.5.23-MariaDB-log
        
        exec() Available:                        yes
        Imagick Available:                       1
        Imagick Enabled:                         1
        Imagick Version:                         1690
        GD Version:                              bundled (2.1.0 compatible)
        Number of foreign key:                   12 found.
     
    Config Information
    ------------------
        version:                                 050102
        check_for_updates:                       0
        sorting_photos_col:                      title
        sorting_photos_order:                    ASC
        sorting_albums_col:                      title
        sorting_albums_order:                    ASC
        imagick:                                 1
        skip_duplicates:                         1
        small_max_width:                         0
        small_max_height:                        360
        medium_max_width:                        1920
        medium_max_height:                       1080
        lang:                                    en
        image_overlay_type:                      none
        default_license:                         none
        compression_quality:                     90
        grants_full_photo_access:                1
        delete_imported:                         0
        mod_frame_enabled:                       1
        mod_frame_refresh:                       30
        thumb_2x:                                1
        small_2x:                                1
        medium_2x:                               1
        landing_page_enable:                     0
        site_owner:                              
        landing_title:                           John Smith
        landing_subtitle:                        Cats, Dogs & Humans Photography
        sm_facebook_url:                         https://www.facebook.com/JohnSmith
        sm_flickr_url:                           https://www.flickr.com/JohnSmith
        sm_twitter_url:                          https://www.twitter.com/JohnSmith
        sm_instagram_url:                        https://instagram.com/JohnSmith
        sm_youtube_url:                          https://www.youtube.com/JohnSmith
        landing_background:                      dist/cat.webp
        site_title:                              
        footer_show_copyright:                   0
        site_copyright_begin:                    2019
        site_copyright_end:                      2019
        footer_additional_text:                  
        footer_show_social_media:                0
        search_public:                           0
        SL_enable:                               0
        SL_for_admin:                            0
        recent_age:                              1
        grants_download:                         1
        photos_wraparound:                       1
        map_display:                             0
        zip64:                                   1
        map_display_public:                      0
        map_provider:                            Wikimedia
        force_32bit_ids:                         0
        map_include_subalbums:                   0
        update_check_every_days:                 3
        has_exiftool:                            0
        share_button_visible:                    0
        import_via_symlink:                      0
        has_ffmpeg:                              0
        location_decoding:                       0
        location_decoding_timeout:               30
        location_show:                           1
        location_show_public:                    0
        rss_enable:                              0
        rss_recent_days:                         7
        rss_max_items:                           100
        prefer_available_xmp_metadata:           0
        editor_enabled:                          1
        lossless_optimization:                   0
        swipe_tolerance_x:                       150
        swipe_tolerance_y:                       250
        local_takestamp_video_formats:           .avi|.mov
        log_max_num_line:                        1000
        unlock_password_photos_with_url_param:   0
        nsfw_visible:                            1
        nsfw_blur:                               0
        nsfw_warning:                            0
        nsfw_warning_admin:                      0
        nsfw_banner_override:                    
        map_display_direction:                   1
        album_subtitle_type:                     oldstyle
        upload_processing_limit:                 1
        new_photos_notification:                 0
        legacy_id_redirection:                   1
        zip_deflate_level:                       6
        SA_enabled:                              0
        default_album_protection:                1
        album_decoration:                        all
        album_decoration_orientation:            row
        allow_username_change:                   1
        auto_fix_orientation:                    0
        use_job_queues:                          1
        random_album_id:                         starred
        use_last_modified_date_when_no_exif_date: 0
        ffmpeg_path:                             
        ffprobe_path:                            
        layout:                                  justified
        date_format_photo_thumb:                 M j, Y, g:i:s A e
        date_format_photo_overlay:               M j, Y, g:i:s A e
        date_format_sidebar_uploaded:            M j, Y, g:i:s A e
        date_format_sidebar_taken_at:            M j, Y, g:i:s A e
        date_format_hero_min_max:                F Y
        date_format_hero_created_at:             M j, Y, g:i:s A T
        date_format_album_thumb:                 M Y
        upload_chunk_size:                       0
        nsfw_banner_blur_backdrop:               0
        search_pagination_limit:                 1000
        search_minimum_length_required:          4
        photo_layout_justified_row_height:       320
        photo_layout_masonry_column_width:       300
        photo_layout_grid_column_width:          250
        photo_layout_square_column_width:        200
        photo_layout_gap:                        12
        display_thumb_album_overlay:             always
        display_thumb_photo_overlay:             hover
        default_album_thumb_aspect_ratio:        1/1
        use_album_compact_header:                0
        login_button_position:                   left
        back_button_enabled:                     0
        back_button_text:                        Return to Home
        back_button_url:                         /
```

### Browser and system
chrome 121
Windows 11

### PR DESCRIPTION
**Pull request by ildyria**
Fixes #2129
Fixes #2151
Fixes #2207
Fixes #2247

### CODE REVIEW

File: resources/js/lycheeOrg/actions/swipeActions.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		const target = document.getElementsByTagName("body")[0];
+		this.gesture = new TinyGesture(target, options);
+		this.listeners = [];
+	}
+
+	register(view: AlbumView) {
+		this.listeners.push(
+			this.gesture.on("swipeleft", () => {
+				if (view.photo_id !== null) {
+					console.log("swipeleft");
```
Comment (d7415) [inline resources/js/lycheeOrg/actions/swipeActions.ts:51]: Do we still want these?

File: resources/js/lycheeOrg/actions/swipeActions.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+					view.previous();
+				}
+			}),
+		);
+		this.listeners.push(
+			this.gesture.on("swiperight", () => {
+				if (view.photo_id !== null) {
+					console.log("swiperight");
```
Comment (d7415) [inline resources/js/lycheeOrg/actions/swipeActions.ts:59]: As above

File: resources/js/lycheeOrg/actions/swipeActions.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+					view.next();
+				}
+			}),
+		);
+		this.listeners.push(
+			this.gesture.on("swipeup", () => {
+				if (view.photo_id !== null) {
+					console.log("swipeup");
```
Comment (d7415) [inline resources/js/lycheeOrg/actions/swipeActions.ts:67]: And this one

**Review by d7415 — APPROVED** [review]
Untested, but I don't see anything problematic.

File: resources/js/lycheeOrg/actions/swipeActions.ts (hunk 1 above)
Comment (ildyria) [reply resources/js/lycheeOrg/actions/swipeActions.ts:51]: you mean the log ?

Comment (ildyria) [reply resources/js/lycheeOrg/actions/swipeActions.ts:51]: I would keep them for now, just to make sure, they can be removed later. :)

**Pull request merged**